lib/util/sdp.js

```diff
@@ -5,6 +5,31 @@
 var flatMap = require('./').flatMap;
 var guessBrowser = require('./').guessBrowser;
 
+// NOTE(mmalavalli): We cache Chrome's sdpSemantics support in order to prevent
+// instantiation of more than one RTCPeerConnection.
+var isSdpSemanticsSupported;
+
+/**
+ * Check if Chrome supports specifying sdpSemantics for an RTCPeerConnection.
+ * @return {boolean}
+ */
+function checkIfSdpSemanticsIsSupported() {
```

Comment:
I think it would be bit easier to read if we split into two functions
```
 function _checkIfSdpSemanticsIsSupported() {
     if (typeof RTCPeerConnection === 'undefined') {
        return false;;
     }
    try {
      new RTCPeerConnection({ sdpSemantics: 'foo' });
      return false;
    } catch (e) {
      return true;
    }
}
 
function checkIfSdpSemanticsIsSupported() {
  if (isSdpSemanticsSupported !== null) {
    isSdpSemanticsSupported = _checkIfSdpSemanticsIsSupported();
  }
  return isSdpSemanticsSupported;
}
```

Reply:
@makarandp0 Normally I would agree, but this is a small function.

Reply:
This is open source, and lot of people would be reading this code, But your call.